src/components/catalog/UserCatalog.jsx

```diff
@@ -0,0 +1,45 @@
+import {Button, Table} from "reactstrap";
+import useFetchCatalogItems from "./fetchCatalogItems";
+
+function UserCatalog() {
+
+    const data = useFetchCatalogItems();
```

Comment:
we can use our old process of getting data.

Reply:
ok